.github/workflows/mirrormaker_offset_tool_release.yml

```diff
@@ -0,0 +1,42 @@
+# The workflow to create PRs with release commits.
+name: Create Mirrormaker Offset Sync Inspector release
+on:
+  push:
+    tags:
+      - mm2-offset-sync-inspector-*
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Setup Java SDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: 11
+          distribution: "temurin"
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.commit_hash }}
```

Comment:
Is this working? Isn't this a custom input from missing workflow_dispatch?

Reply:
Leftover, removed.